lib/cli.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+
+const path = require("path");
+const watcher = require("./index");
+
+function usage() {
+  console.log("Usage: watcher <pattern> [<pattern>...] [options]");
+  console.log("  -h, --help\tShow help");
+  console.log("  -v, --verbose\tMake output more verbose");
+}
+
+function start(dirs, verbose) {
+  const options = { recursive: true };
```

Comment:
Were you meaning to make this another user-controlled flag?

Reply:
No, I think that `recursive` is probably the wanted option here. Do you think I should use an option here?